app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -62,9 +62,13 @@ def textual_physical_servers
     h
   end
 
-  def textual_hosts
+  def textual_physical_servers_with_host
     count_of_host_relationships = (@record.physical_servers.select { |server| !server.host.nil? }).length
-    {:label => _("Hosts"), :icon => "pficon pficon-screen", :value => count_of_host_relationships}
+    h = {:label => _("Physical Servers with Host"), :icon => "pficon pficon-server", :value => count_of_host_relationships}
+    if count_of_host_relationships > 0
+      h[:link] = "/ems_physical_infra/#{@ems.id}?display=physical_servers&options=physical_servers_with_host"
```

Comment:
@martinpovolny Created https://github.com/ManageIQ/manageiq-ui-classic/pull/1553/commits/e6515c9673597b5e1ee9eeaf4e3c98161c7663c5 to address your above comment https://github.com/ManageIQ/manageiq-ui-classic/pull/1553#discussion_r123097075.

Trying to keep the PR up-to-date as much as possible by addressing the feedback that I can address right away.
I'm aware, we still have to worry about the whole `session`/`report_data`/`get_view` thing.